po/ChangeLog

```diff
@@ -1,3 +1,14 @@
+2016-07-24  gettextize  <bug-gnu-gettext@gnu.org>
```

Comment:
None of these changes to the po/ directory should be committed. Your commits change about 60K lines and at a cursor glance, most appear to be in the po/ directory. Let the language translators make the changes to this directory.